components/src/components/hubspot-form/hubspot-form.tsx

```diff
@@ -46,6 +46,39 @@ export class HubspotForm {
             });
             this.isLoading = false;
             this.didLoad = true;
+
+            // Hidden form fields from HubSpot created forms causes gaps to appear in
+            // the form which is less then desireable. So we need to find the hidden
```

Comment:
```suggestion
            // the form which is less then desirable. So we need to find the hidden
```

Still, rather that do this with code, could we do it instead with some targeted CSS?

Reply:
Ah, perhaps not, if it targets a parent.

Reply:
Yeah I tried a CSS approach first, but the way HubSpot applies the `hidden` style makes it I think impossible to currently in just pure CSS.